### PR TITLE
fix typo racing

### PR DESCRIPTION
**Pull request by coco875**

### CODE REVIEW

**Comment by coco875** [comment]
don't match

File: src/racing/memory.c [hunk 1]
```diff
@@ -243,9 +243,9 @@ UNUSED u32 main_pool_pop_state(void) {
 // similar to sm64 dma_read
 UNUSED void *func_802A80B0(u8 *dest, u8 *srcStart, u8 *srcEnd) {
     void *addr;
-    u32 size = srcStart - dest;
```
Comment (MegaMech) [inline src/racing/memory.c:246]: ptrdiff_t

File: src/racing/memory.c [hunk 2]
```diff
@@ -507,7 +507,7 @@ void unpack_displaylist(Gfx *arg0, u8 *args, UNUSED s8 opcode) {
 
 // end displaylist
 void unpack_end_displaylist(Gfx *arg0, UNUSED u8 *arg1, UNUSED s8 arg2) {
-    arg0[sGfxSeekPosition].words.w0 = G_ENDDL << 24;
+    arg0[sGfxSeekPosition].words.w0 = (uintptr_t)(uint8_t)G_ENDDL << 24;
```
Comment (MegaMech) [inline src/racing/memory.c:510]: Does this match?

File: src/racing/memory.c [hunk 3]
```diff
@@ -938,7 +939,7 @@ void unpack_spline_3D(Gfx *gfx, u8 *arg1, UNUSED s8 arg2) {
         temp_v0 = arg1[sPackedSeekPosition++];
         phi_a0 |= (temp_v0 & 0xF) * 2;
     }
-    gfx[sGfxSeekPosition].words.w0 = (G_LINE3D << 24);
+    gfx[sGfxSeekPosition].words.w0 = ((uintptr_t)(uint8_t)G_LINE3D << 24);
     gfx[sGfxSeekPosition].words.w1 =
```
Comment (MegaMech) [inline src/racing/memory.c:943]: These two lilnes should probably say G_QUAD not G_LINE3D as line3d only exists in the line3d microcode and not in f3dex.

**Review by MegaMech — CHANGES_REQUESTED** [review]

File: src/racing/memory.c (hunk 2 above)
Comment (coco875) [reply src/racing/memory.c:510]: yes

File: src/racing/memory.c (hunk 1 above)
Comment (coco875) [reply src/racing/memory.c:246]: not match with ptrdiff_t

File: src/racing/memory.c [hunk 4]
```diff
@@ -701,20 +701,20 @@ void unpack_tile_sync(Gfx *gfx, u8 *args, s8 opcode) {
...
-    u32 tile;
+    uintptr_t arg;
+    uintptr_t lo;
+    uintptr_t hi;
+    uintptr_t addr;
+    uintptr_t width;
+    uintptr_t height;
+    uintptr_t fmt;
+    uintptr_t siz;
+    uintptr_t tmem;
```
Comment (MegaMech) [inline src/racing/memory.c:716]: technically these all shouldn't be uintpr_t but whatevs

**Review by MegaMech — APPROVED** [review]

**Pull request merged**